League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -143,6 +156,15 @@ private void startCloningRepositories()
                 if (!Directory.Exists(cloningPath))
                 {
                     Directory.CreateDirectory(cloningPath);
+                }
+                if (Directory.Exists(cloningPath))
+                {
+                    System.IO.DirectoryInfo cloningDir = new System.IO.DirectoryInfo(cloningPath);
+                    // Set attributes in cloningDir to normal, as git files refuse to be deleted
+                    setAttributesNormal(cloningDir);
+                    cloningDir.Delete(true);
```

Comment:
I suggest to ask the user with a warning message box, something like `The folder {folderName} is already in use, are you sure you want to delete existing files?`.

Another possibility is just to create a new folder `GameServer2` and advance the number if the folder is in use.